test/command_line/test_integrate.py

```diff
@@ -26,6 +26,7 @@ def test2(dials_data, tmpdir):
             "modified_input.json",
             "profile.fitting=False",
             "integration.integrator=3d",
+            "output_unintegrated_reflections=True",
```

Comment:
I'd prefer not adding this everywhere where it isn't needed (because of the default) - it's verbose, and makes it harder to change/remove/rename

Reply:
Done